l10n_nl_postcode/__manifest__.py

```diff
@@ -3,12 +3,11 @@
 
 {
     "name": "Dutch postcode validation for Partners",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.0",
     "category": "Localization",
     "author": "Onestein,Odoo Community Association (OCA)",
     "maintainers": ["astirpe"],
     "website": "https://github.com/OCA/l10n-netherlands",
     "license": "AGPL-3",
     "depends": ["base"],
-    "external_dependencies": {"python": ["python-stdnum"]},
```

Comment:
Why is this dependency removed?

Reply:
It's already in standard: https://github.com/odoo/odoo/blob/16.0/requirements.txt#L36

Reply:
I know, it is a core requirement. Yet, to my understanding it is OCA best practice to still include it in the modules manifest?
Either way, non blocking.